graal-js/src/com.oracle.truffle.js.test.fetch/js/fetch-globals.js

```diff
@@ -40,7 +40,7 @@ const url = "http://localhost:8080";
     assertSame('[object Response]', new Response().toString());
 })();
 
-(function shouldSupportProperStringOutput() {
+(function shouldThrowWithoutInitOptionObject() {
```

Comment:
Thanks for pointing out this copy/paste bug. The name that you suggest is already much better than the old name but maybe consider something like `shouldThrowWithInvalidInitOptions[Argument]`. The name that you have suggested implies that the built-ins should throw when `init/options` argument is not present, but that's not correct. It is fine if the argument is not present at all.

Reply:
Yes shouldThrowWithInvalidArguments is better, we could of course say what it does (checking for rejection of integers), but I guess it’s fine this way. Should I maybe add rejected string (and a new test allowing null and {}?